test.js

```diff
@@ -914,3 +915,45 @@ describe('Elastic()', () => {
     expect(clientInfo.statusCode).toBe(200);
   });
 });
+
+describe('Fetch()', () => {
+  test('Fetch(url) should return 200 on success with a GET request', async (done) => {
+    const url = 'https://www.google.ca';
```

Comment:
Let's not have our tests depend on external URLs.  You can use nock as a dev dependency to simulate responses.